src/custom_conf_div.yaml

```diff
@@ -1,16 +1,19 @@
 contract_observation_clause: ct
 contract_execution_clause:
 # - seq
+# - bottom
```

Comment:
This config file is unlikely to be useful for anything besides debugging VSPEC. I think it's better not to commit it to the repo.